### PR TITLE
[CAM-11111] provider synchronization process details

### PR DESCRIPTION
**Pull request by DiegoSanjuan**
@gsanchezu

### CODE REVIEW

File: Cloud Application Manager/Core Concepts/providers.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {{{
 "title": "Providers",
 "date": "10-17-2018",
```
Comment (gsanchezu) [inline Cloud Application Manager/Core Concepts/providers.md]: Please update date

File: Cloud Application Manager/Core Concepts/providers.md [hunk 2]
```diff
@@ -31,6 +33,40 @@ An active Cloud Application Manager account.
...
+* Configure IAM
+* Secure account
+* Sync ALM
+* Analyze
+* Monitoring
+
+All detailed events and any issue detected during synchronization process will be logged and you can audit the logs of the complete process in logs tab of the provider.
+
+
 #### Supported provider types
```
Comment (gsanchezu) [inline Cloud Application Manager/Core Concepts/providers.md]: Use level 3 title

File: Cloud Application Manager/Core Concepts/providers.md [hunk 3]
```diff
@@ -31,6 +33,40 @@ An active Cloud Application Manager account.
...
+* Available autoscalability configuration (i.e. resource limits or alarms)
+* Allowed type/size of virtual machines
+* Available images or templates to be used at provision time. These are listed in configuration of your provider.
+* Available public keys to secure provisioned virtual machines
+* Available load balancers and target groups
+* Unregistered virtual machines and other cloud resources which are already provisioned in your cloud account in order to allow you to register those virtual machines or resources if you want to do so. These are listed in Unregistered tab of your provider. Discovery of these unregistered resources can be done atomatically if you selected so in your provider Services tab: 'Automatic Discovery of Resources'
+
+You can request Cloud Application Manager to update provider's metadata clicking on Sync button when needed. i.e. after adding new images or firewall rules in your provider account, you need to synchronize your provider in order to select one of newly added resources.
+
+All synchronization process is logged and you can audit it in Logs tab o provider.
```
Comment (gsanchezu) [inline Cloud Application Manager/Core Concepts/providers.md]:
```suggestion
All synchronization process is logged and you can audit it in the **Logs** tab of the provider details page.
```

File: Cloud Application Manager/Core Concepts/providers.md [hunk 4]
```diff
@@ -31,6 +33,40 @@ An active Cloud Application Manager account.
...
+This metadata includes required infrastructure data as follows:
+* Available regions or datacenters
+* Available networks and subnetworks
+* Allowed connectivity options with public or private ip and firewall rules
+* Available autoscalability configuration (i.e. resource limits or alarms)
+* Allowed type/size of virtual machines
+* Available images or templates to be used at provision time. These are listed in configuration of your provider.
+* Available public keys to secure provisioned virtual machines
+* Available load balancers and target groups
+* Unregistered virtual machines and other cloud resources which are already provisioned in your cloud account in order to allow you to register those virtual machines or resources if you want to do so. These are listed in Unregistered tab of your provider. Discovery of these unregistered resources can be done atomatically if you selected so in your provider Services tab: 'Automatic Discovery of Resources'
```
Comment (gsanchezu) [inline Cloud Application Manager/Core Concepts/providers.md]: I would rather specify that the "registrable" resources are VMs and scaling groups, not any resource. Those are the ones displayed under "Unregistered resources" tab. But the rest of the native resources are listed in the Instances page (for AWS and Azure for now)
Please, also add a link to 'Automatic Discovery of Resources' that is explained in AWS or Azure providers

**Comment by gsanchezu** [comment]
Hey @DiegoSanjuan, please address my comments above

File: Cloud Application Manager/Core Concepts/providers.md (hunk 1 above)
Comment (DiegoSanjuan) [reply Cloud Application Manager/Core Concepts/providers.md]: done

File: Cloud Application Manager/Core Concepts/providers.md (hunk 2 above)
Comment (DiegoSanjuan) [reply Cloud Application Manager/Core Concepts/providers.md]: done

File: Cloud Application Manager/Core Concepts/providers.md (hunk 4 above)
Comment (DiegoSanjuan) [reply Cloud Application Manager/Core Concepts/providers.md]: done

File: Cloud Application Manager/Core Concepts/providers.md [hunk 5]
```diff
@@ -45,8 +55,45 @@ Every provider in the list have a gear icon from which you can synchronize or de
...
+This metadata includes required infrastructure data as follows:
+* Available regions or datacenters
+* Available networks and subnetworks
+* Allowed connectivity options with public or private ip and firewall rules
+* Available autoscalability configuration (i.e. resource limits or alarms)
+* Allowed type/size of virtual machines
+* Available images or templates to be used at provision time. These are listed in configuration of your provider
+* Available public keys to secure provisioned virtual machines
+* Available load balancers and target groups
+* Unregistered virtual machines and autoscaling groups which are already provisioned in your cloud account in order to allow you to register them if you want to do so. These are listed in Unregistered tab of your provider. Discovery of these unregistered virtual machines can be done atomatically if you selected so in your provider Services tab: [Automatic Discovery of Resources](https://www.ctl.io/knowledge-base/cloud-application-manager/getting-started/register-existing-instance/#discovering-the-unregistered-instances)
```
Comment (gsanchezu) [inline Cloud Application Manager/Core Concepts/providers.md]: I would also add that other native resources are discovered and displayed in the Instances page with some basic information.

Comment (DiegoSanjuan) [reply Cloud Application Manager/Core Concepts/providers.md]: done

**Pull request merged**